explore_utils.py

```diff
@@ -0,0 +1,17 @@
+def find_most_visit(dataset):
+    """Find what is the most visited times of single customer.(only in train set)
+
+    args:
+        dataset (Dataset): the google analytics dataset.
+
+    returns:
+	    The most visited times in trainset.
+    """
+
+	train_df = dataset.train.copy()
+
+    train_df['transactionRevenue'] = train_df['transactionRevenue'].astype(float)
```

Comment:
Is this line needed? transactionRevenue isn't used anywhere else.

Reply:
Sorry, got several mistakes when rewriting code from my jupyter notebook.
Will change it soon.